package.json

```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "bench": "node ./bench/performances.js",
     "unit-test": "jest --config=./tests/unit.js",
-    "integration-test": "jest --config=./tests/integration.js",
+    "integration-test-mobile": "TEST_DEVICE=mobile jest --config=./tests/integration.js",
+    "integration-test-desktop": "TEST_DEVICE=desktop jest --config=./tests/integration.js",
+    "integration-test": "npm run integration-test-mobile && npm run integration-test-desktop",
```

Comment:
I guess we could event run the mobile and desktop tests in parallel ?

Reply:
Maybe, but we have to be careful not breaking the behavior of the CI with return values.
Let's see later if this brings something.